feat: Unity native support page (#4050)
getsentry/sentry-docs: src/platforms/unity/configuration/native-support.mdx

---
title: Native Support

> description: "Learn how the SDK handles native support."

Reviewer: ```suggestion
description: "Learn how the Unity SDK handles native support."
```

sidebar_order: 30
---

Our Unity SDK currently provides native support for iOS and is enabled by default. It does this by embedding the [Cocoa SDK](/platforms/apple/guides/ios/) inside the generated Xcode project and passing the options to it at build time.

> To opt-out simply uncheck `iOS Native Support` in the configuration window or disable it via:

Reviewer: ```suggestion
To opt out, uncheck `iOS Native Support` in the configuration window or disable it as shown below:
```
